Source/Core/DolphinQt/Config/Graphics/BalloonTip.cpp

```diff
@@ -0,0 +1,299 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt/Config/Graphics/BalloonTip.h"
+
+#include <QBitmap>
+#include <QGraphicsEffect>
+#include <QGraphicsView>
+#include <QGridLayout>
+#include <QGuiApplication>
+#include <QLabel>
+#include <QPainter>
+#include <QPainterPath>
+#include <QPropertyAnimation>
+#include <QPushButton>
+#include <QStyle>
+
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+#include <QScreen>
+#else
+#include <QApplication>
+#include <QDesktopWidget>
+#endif
+
+namespace
+{
+static BalloonTip* theSolitaryBalloonTip = nullptr;
```

Comment:
`snake_case`

This should be wrapped into a unique_ptr, so it doesn't get potentially flagged as a memory leak by tools like asan.